config/environments/development.rb

```diff
@@ -67,7 +67,7 @@
   config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  # config.action_view.annotate_rendered_view_with_filenames = false
```

Comment:
Why do we need this?

Reply:
I'm just enabling it locally to see where the code for the particular page comes from.